tests/db/migration_tests/test_v202212010_to_v202301100.py

```diff
@@ -38,8 +38,5 @@ async def test_added_environment_metrics_tables(
     await migrate_db_from()
 
     for table_name in ["environmentmetricsgauge", "environmentmetricstimer"]:
-        assert "grouped_by" in (await get_columns_in_db_table(table_name))
-
         columns_pk = await get_primary_key_columns_in_db_table(table_name)
         assert len(columns_pk) == 4
-        assert "grouped_by" in columns_pk
```

Comment:
@wouterdb FYI: We decided to rename a column in the metrics tables from "grouped_by" to "category" which broke the migration test where this column was added. I removed the assertion for this reason.

Reply:
(This old migration script has never been released and will only be released together with this new one.)